src/OrleansTestingHost/OrleansTestingHost.csproj

```diff
@@ -64,6 +64,10 @@
     <Compile Include="TestingUtils.cs" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\OrleansManager\OrleansManager.csproj">
```

Comment:
are we sure we absolutely sure we want this dependency and deal with side effects in nuget and so on? What's interesting about that DLL? It's a console app. Haven't read yet how it's used, but commenting here first as maybe it was not by design